Medium/secrets.py

```diff
@@ -0,0 +1,3 @@
+SLACK_SIGNING_SECRET = "eb8895a372a2c0e6c37c0ef88eeb6bde"
```

Comment:
Delete this file.

Reply:
Pushed by mistake! On  it

Reply:
@shubhank-saxena That file shouldn't actually be there cause i had ignored it, but it may have been pushed due to conflicts in the .gitignore file. Though I've deleted the file. Ready for review again!